tests/testthat/test-condition_message_linter.R

```diff
@@ -38,6 +44,25 @@ test_that("condition_message_linter blocks simple disallowed usages", {
     condition_message_linter()
   )
 
+  # `sep` argument allowed, but only if it is different from default
+  expect_lint(
+    "stop(paste(x, sep = ' '))",
+    rex::rex("Don't use paste to build stop strings."),
+    condition_message_linter()
+  )
+
+  expect_lint(
```

Comment:
i don't think we need three tests for this, unless it's being done with a `patrick` parameterized test

Reply:
I have added parametrized tests now.